dask/dataframe/dispatch.py

```diff
@@ -47,6 +47,9 @@ def concat(
         Whether to ignore the order when doing the union of categoricals.
         Default is False.
     """
+
+    print("~~ James was here ~~")
```

Comment:
Should this be removed ?

Reply:
Yeah, definitely. I've just temporarily added this to try and trigger the automatic `dispatch` label